README.md

```diff
@@ -71,6 +65,37 @@ For all explanations of our available input parameters and output variables, see
 
 See also our [example recipes][recipes-doc].
 
+## Used By
+
+<p align="center">
+  <a href="https://github.com/Microsoft"><img src="https://avatars.githubusercontent.com/u/6154722?s=200&v=4" alt="Microsoft" width="28"/></a>
+  <strong>Microsoft</strong>&nbsp;&nbsp;
+  <a href="https://github.com/apache"><img src="https://avatars.githubusercontent.com/u/47359?s=200&v=4" alt="Apache" width="28"/></a>
+  <strong>Apache</strong>&nbsp;&nbsp;
+  <a href="https://github.com/nasa"><img src="https://avatars.githubusercontent.com/u/848102?s=200&v=4" alt="NASA" width="28"/></a>
+  <strong>NASA</strong>&nbsp;&nbsp;
+  <a href="https://github.com/samsung"><img src="https://avatars.githubusercontent.com/u/6210390?s=200&v=4" alt="Samsung" width="28"/></a>
+  <strong>Samsung</strong>&nbsp;&nbsp;
+  <a href="https://github.com/TheAlgorithms"><img src="https://avatars.githubusercontent.com/u/20487725?s=200&v=4" alt="TheAlgorithms" width="28"/></a>
+  <strong>TheAlgorithms</strong>&nbsp;&nbsp;
+  <a href="https://github.com/CachyOS"><img src="https://avatars.githubusercontent.com/u/85452089?s=200&v=4" alt="CachyOS" width="28"/></a>
+  <strong>CachyOS</strong>&nbsp;&nbsp;
+  <a href="https://github.com/nextcloud"><img src="https://avatars.githubusercontent.com/u/19211038?s=200&v=4" alt="Nextcloud" width="28"/></a>
+  <strong>Nextcloud</strong>&nbsp;&nbsp;
```

Comment:
There should be a line break or something here to ensure the logo is next to the name.
![image](https://github.com/user-attachments/assets/75110f57-4230-459e-993a-3e2e9e905a27)

Notice in the picture, the NNStreamer logo is on the first line, but the "NNStreamer" text is on the second line.

Reply:
Still happening with a different brand now:
![image](https://github.com/user-attachments/assets/8d39c1ab-7b73-4faa-b97b-f23bce1ad6ae)

And for some reason the logo images are larger